In [2]:
import pandas as pd
import numpy as np

# Marotta e gli stadi in Italia

In questo python notebook cercheremo di analizzare la seguente frase del dirigente sportivo dell'Inter Marotta:

> Questa è la nota dolente, i numeri contano per fare delle riflessioni. Negli ultimi 20 anni, in Europa, sono stati costruiti 250 nuovi stadi, di questi solo 5 in Italia. Questo la dice lunga su quanto siamo fanalino di coda

Se partiamo dal presupposto che l'intento di Marotta sia quello di evidenziare una carenza di nuovi stadi in Italia, e quindi usa questa frase per provarlo, possiamo cercare di capire se ha ragione o meno usando solo la frase stessa.

## Livello 1: 250, 5, 20 e l'Europa

Il primo livello di analisi utilizza, come informazione, solo quella contenuta nella frase di Marotta, ossia 250 stadi in 20 anni in Europa e 5 in Italia. Se l'intento di Marotta e' quello di concludere che in Italia si costruiscono pochi stadi, allora possiamo calcolare se cinque nuovi stadi in Italia, rispetto ai rimanenti 245 nel resto d'Europa, siano, in maniera significativa, statisticamente molto al di sotto della media Europea. 

Dato che Marotta parla in generale di Europa, consideriamo prima tre diverse ipotesi su cosa intendesse per Europa. Dato che non specifica a cosa si possa riferire quando parla di Europa, prendiamo come prima istanza 3 diverse definizioni di Europa per calcolare quante nazioni, in totale, sono nel continente:
- Europa come EU
- Europa come continente
- Europa come UEFA

A seconda delle definizioni di Europa, cambia il numero di nazioni che ne fanno parte e questo influenza la media di nuovi stadi per ogni paese: 

$$
\overline{nuovi\_stadi} = \frac{250}{k}
$$

dove $k$ rappresenta il numero di nazioni della definizione di Europa. 

## Europa come EU

Nel caso in cui vogliamo considerare Europa come gli attuali stati che formano l'Unione Europea, allora $k = 27$ e quindi avremo una media attesa di:

$$
\mu = \frac{250}{27} \approx 9.26
$$

In [3]:
print(f"mu = {np.round(250/27), 2}")

mu = (np.float64(9.0), 2)


L'Italia, a quanto dice Marotta, negli ultimi 20 anni ha costruito 5 stadi, quindi la differenza con la media e' di:

$$
x - \mu = 5 -9.26 = -4.26
$$

e quindi, in Italia, ha circa 4.26 in meno rispetto alla media Europea. E' abbastanza per dire che siamo sotto la media europea? Supponiamo che il numero di stadi per ogni nazione europea segua un modello di Poisson (ossia, esiste la stessa probabilita' di costruire nuovi stadi in ogni nazione in maniera indipendente), quindi ogni nuovo stadio ha probabilita' 

$$
p = \frac{1}{k} = \frac{1}{27}
$$

di essere costruito in quella nazione, allora il numero di stadi per ogni nazione risulta:

$$
X \sim \text{Binomial}(n = 250, p = 1/27)
$$

E dato che $n$ e' grande mentre $p$ e' relativamente piccolo, allora possiamo usare un'approssimazione di Poisson:

$$
X \sim Poisson(\lambda = np = 9.26)
$$

Possiamo ora rispondere alla domanda: qual'e' la probabilita' di osservare la costruzione di 5 o meno di 5 stadi:

$$
P( X \le 5 | \lambda = 9.26)
$$

In [5]:
from scipy.stats import poisson

n = 250
p = 1/27
lambda_ = n*p
p_value = poisson.cdf(5, lambda_)
print(f"the resulting p-value is {p_value}")

the resulting p-value is 0.10082956070176977


Abbiamo quindi un p-value di circa 0.101 e quindi possiamo concludere che: 
> Considerando il numero di nazioni all'interno dell'Europa come le nazioni dell'Unione Europea e sotto l'ipotesi di un modello dove ogni nazione ha la stessa probabilita' di costruire un nuovo stadio negli ultimi 20 anni, il fatto che l'Italia ne abbia costruiti di meno non e' statisticamente significativo, ossia non e' inusuale

Riassumendo:
- media attesa: 9.26
- media osservata: 5
- differenza: -4.26
- z-score (sotto approssimazione di Poisson):
$$
z = \frac{5-9.26}{\sqrt{9,26}} = -1.4
$$
- p-value: 0.102

E quindi l'Italia e' sotto la media, ma non in maniera significativa (ossia, dal punto di vista strettamente probabilistico, la spiegazione puo' essere semplicemente data dal caso)